jenkins/Dockerfile-blossom.ubuntu

```diff
@@ -25,7 +25,7 @@
 
 ARG CUDA_VER=11.0
 ARG UBUNTU_VER=18.04
-ARG UCX_VER=v1.11.0
+ARG UCX_VER=v1.11.2
```

Comment:
Should this be updated to not require the leading `v` to be consistent with the other Docker examples?

Reply:
Sure, I can do that.